README.md

```diff
@@ -1,3 +1,9 @@
+# Status: Archived
+This repository has been archived and is no longer maintained.
+Use official [metafacture-core](https://github.com/metafacture/metafacture-core) for core functionality and have a look at [lobid-resources](https://github.com/hbz/lobid-resources/) for some enhancements which are not (yet) part of metafacture-core.
+
+![status: inactive](https://img.shields.io/badge/status-inactive-red.svg)
```

Comment:
Looking at https://github.com/hbz/metafacture-core/blob/3e1ada72241915cd0eda08bd52fc8924bff80f5d/README.md my initial impression is 'Metafacture is inactive' (because of the color and the big logo I guess). Maybe we could just remove the badge:

```suggestion
```

Reply:
Copied that from https://github.com/googlearchive/examples/edit/master/README.md.
It's the fork that is going to be inactive. 
Updated the README to make this clearer.